sceptre/environment.py

```diff
@@ -74,20 +72,25 @@ def __repr__(self):
         )
 
     @staticmethod
-    def _check_env_path_valid(path):
+    def _validate_path(path):
         """
+        Converts backslashes to forward slashes.
         Raises an InvalidEnvironmentPathError if the path has a leading or
         trailing slash.
 
         :param path: A directory path
         :type path: str
         :raises: sceptre.exceptions.InvalidEnvironmentPathError
+        :returns: A normalised path with forward slashes.
+        :returns: string
         """
+        path = path.replace('\\', '/')
```

Comment:
So above you are using a string literal and should be aware that you haven't escaped your backslashes. When paths for Windows are read by python - they will have escaped backslashes e.g. "\\a\\n\\c".